### PR TITLE
hub content quick fixes

### PR DESCRIPTION
**Pull request by JohnWattsGDS**
Changes to hub content.

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -146,38 +146,46 @@ en:
...
+      no_charge: You don't need to pay to use GOV.UK Verify.
       summary: How companies can verify identities
       details_html: |
-        <p>These companies can use their own data, and they can access data like credit records. They’re able to check that information from passports and driving licences is correct.</p>
-        <p>Certified companies meet government privacy standards, so you can trust them with your personal information. They won’t share or sell your data for any other purposes without your consent.</p>
-        <p>There won’t be any effect on your credit score.</p>
-        <p>More than one certified company means you choose who verifies your identity, and it means the system overall is stronger.</p>
+        <p>These companies can check it's really you by using their own records, or by accessing information like credit records. If you have a passport or driving licence, they can check that information too.</p>
+        <p>These 5 companies meet government privacy standards, so you can trust them with your personal information. They won’t share or sell your information for any other purpose without your consent.</p>
```
Comment (bjgill) [inline config/locales/en.yml]: > `5`

Have we documented this somewhere so that we know to change this number when the number of IDPs increases/decreases?

Comment (JohnWattsGDS) [reply config/locales/en.yml]: It's a content thing, the content people take care of content when the content changes (for example, if the number of IDPs changes).

Comment (bjgill) [reply config/locales/en.yml]: OK, so we don't need to record it in the IDP disconnection section of the tech manual?

Comment (JohnWattsGDS) [reply config/locales/en.yml]: It may be worth adding for the sake of belts and braces.

**Pull request merged**